jx_sqlite/expressions.py

```diff
@@ -248,11 +248,11 @@ def to_sql(self, schema, not_null=False, boolean=False):
     acc = []
     for t in "bsnj":
         if lhs[t] and rhs[t]:
-            acc.append("(" + lhs[t] + ") = (" + rhs[t] + ")")
+            acc.append( "(" + lhs[t] + ") <> (" + rhs[t] + ") OR ((" + lhs[t] + ") IS NULL)")
```

Comment:
This line is now lacking symmetry:  `{"ne": [a, b]} ⇔ {"ne": [b, a]}`.  I have updated the `test_ne` to match the document